RIGImageGallery/RIGSingleImageViewController.swift

```diff
@@ -8,34 +8,44 @@
 
 import UIKit
 
-class RIGSingleImageViewController: UIViewController {
+open class RIGSingleImageViewController: UIViewController {
 
-    var viewerItem: RIGImageGalleryItem? {
+    open var viewerItem: RIGImageGalleryItem? {
         didSet {
             viewerItemUpdated()
         }
     }
 
-    let scrollView = RIGAutoCenteringScrollView()
+    open let scrollView = RIGAutoCenteringScrollView()
+    open let activityIndicator: UIActivityIndicatorView = {
```

Comment:
good call, I re-implemented the loading to handle constraint updates on didSet, and the move all of the things happening on `loadView` into `viewDidLoad` instead